yt_dlp/postprocessor/sponsorblock.py

```diff
@@ -68,12 +72,13 @@ def duration_filter(s):
 
         def to_chapter(s):
             (start, end), cat = s['segment'], s['category']
+            title = s['description'] if cat == 'chapter' else self.CATEGORIES[cat]
```

Comment:
To clarify, `description` is present if and only if categories is 'chapter', correct? Is there any case where a chapter doesn't have description or where a sponsor has one?

Reply:
The API says "title for chapters, empty string for other segments", so I'll go by that